src/parser/esprima_cpp/esprima.cpp

```diff
@@ -2285,6 +2285,11 @@ class Parser {
             return ScanExpressionResult(ASTNodeType::ArrowParameterPlaceHolder);
         }
 
+        /* handle single rest parameter */
+        if (this->match(PeriodPeriodPeriod)) {
+            this->nextToken();
+        }
```

Comment:
I think that this part couldn't handle the rest parameter entirely. `groupExpression` module in Parser should be fixed based on the esprima. I've updated #322 to enable rest parameter in arrow function. Sorry if you are working on the same issue.

Reply:
Indeed, #322 seems a more general solution for this problem. No problem, I'm closing this issue.